src/eddymotion/utils.py

```diff
@@ -0,0 +1,171 @@
+    - RANDOM: Sorts the items in a random order.
+    - BVALUE: Sorts the items based on their b-value.
+    - CENTRALSYM: Sorts the items based on their central symmetry.
+    """
+    LINEAR = "linear"
+    RANDOM = "random"
+    BVALUE = "bvalue"
+    CENTRALSYM = "centralsym"
+
+
+def sort_dwdata_indices(dwdata, strategy, seed=None):
+    """Sort the DWI data volume indices following the given strategy.
+
+    Parameters
+    ----------
+    dwdata : :obj:`~eddymotion.dmri.DWI`
+        DWI dataset, represented by this tool's internal type.
+    strategy : :obj:`~eddymotion.utils.SortingStrategy`
+        The sorting strategy to be used. Available options are:
+        - SortingStrategy.LINEAR: Sort the indices linearly.
+        - SortingStrategy.RANDOM: Sort the indices randomly.
+        - SortingStrategy.BVALUE: Sort the indices based on the last column of gradients in ascending order.
+        - SortingStrategy.CENTRALSYM: Sort the indices in a central symmetric manner.
+    seed : :obj:`int` or :obj:`bool`, optional
+        Seed the random number generator. If an integer, the value is used to
+        initialize the generator; if ``True``, the arbitrary value
+        of ``20210324`` is used to initialize it.
+
+    Returns
+    -------
+    index_order : :obj:`numpy.ndarray`
+        The sorted index order.
+    """
+    if strategy == SortingStrategy.LINEAR:
+        return linear_action(dwdata)
+    elif strategy == SortingStrategy.RANDOM:
+        return random_action(dwdata, seed)
+    elif strategy == SortingStrategy.BVALUE:
+        return bvalue_action(dwdata)
+    elif strategy == SortingStrategy.CENTRALSYM:
+        return centralsym_action(dwdata)
+    else:
+        raise ValueError("Invalid sorting strategy")
+
+
+def linear_action(dwdata):
+    """
+    Sort the DWI data volume indices linearly
+
+    Parameters:
+    dwdata : :obj:`~eddymotion.dmri.DWI`
+        DWI dataset, represented by this tool's internal type.
+
+    Returns:
+    index_order : :obj:`numpy.ndarray`
+    The sorted index order.
+    """
+    index_order = np.arange(len(dwdata))
+
+    return index_order
+
+
+def random_action(dwdata, seed=None):
+    """Sort the DWI data volume indices.
+
+    Parameters
+    ----------
+    dwdata : :obj:`~eddymotion.dmri.DWI`
+        DWI dataset, represented by this tool's internal type.
+    seed : :obj:`int` or :obj:`bool`, optional
+        Seed the random number generator. If an integer, the value is used to
+        initialize the generator; if ``True``, the arbitrary value
+        of ``20210324`` is used to initialize it.
+
+    Returns
+    -------
+    index_order : :obj:`numpy.ndarray`
+        The sorted index order.
+    """
+
+    _seed = None
+    if seed or seed == 0:
+        _seed = 20210324 if seed is True else seed
+
+    rng = np.random.default_rng(_seed)
+
+    index_order = np.arange(len(dwdata))
+    rng.shuffle(index_order)
+
+    return index_order
```

Comment:
Let's return just lists of integers (I'd prefer these were generators, but for a first stab looks good with a list):

```suggestion
    return index_order.to_list()
```